testing/kuttl/kuttl-test.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestSuite
+testDirs:
+- testing/kuttl/e2e/
+timeout: 120
+parallel: 2
+# comment out namespace if you want kuttl tests to run in its
+# own generated namespace
+namespace: postgres-operator
```

Comment:
Should the default here be to run in multiple namespaces :question: Then have a comment saying to add the line if you want them in a single ns.

Reply:
from dealing with previous e2e testing, I dislike running in generated namespace because it has made debugging issues a little more difficult but I can see in a CI/CD env it being useful to be in generated namespaces. I'll make an update to default to multiple namespaces and add the comment.

Reply:
updated